src/main.rs

```diff
@@ -362,6 +362,11 @@ fn run() -> Result<()> {
         } else {
             ((8 / group_size) * (base_digits * group_size + 1)) + 2
         };
+
+        if terminal_width < offset {
+            return 1;
+        }
```

Comment:
It looks like you could also remove this branch and use `terminal_width.saturating_sub(offset) / col_width < 1` below?

Reply:
Looks like it's only for nightly: https://doc.rust-lang.org/std/intrinsics/fn.saturating_sub.html

Oh it's available for u32 and u64 in stable:

https://doc.rust-lang.org/std/primitive.u32.html#method.saturating_sub
https://doc.rust-lang.org/std/primitive.u64.html#method.saturating_sub

Let check it out. Thanks!

Reply:
Ok, that worked great

<img width="690" alt="image" src="https://github.com/user-attachments/assets/250f6a11-532a-4d5d-aab1-325c00a8fa59" />

`cargo test` works as well.

Time to see what CI does